test-services-docker-compose.yaml

```diff
@@ -13,15 +13,42 @@ services:
     networks:
       - main
 
+  aws:
+    container_name: aws
+    image: localstack/localstack:3.4.0
+    ports:
+      - "127.0.0.1:4566:4566"            # LocalStack Gateway
+      - "127.0.0.1:4510-4559:4510-4559"  # External services port range
+    environment:
+      AWS_DEFAULT_REGION: ${AWS_DEFAULT_REGION:-us-west-2}
+      DEFAULT_REGION: ${AWS_DEFAULT_REGION:-us-west-2}
+      SQS_ENDPOINT_STRATEGY: path
+      SERVICES: sqs
+      LOCALSTACK_HOST: aws
+      DEBUG: 1
+      LS_LOG: trace
+    volumes:
+      - "/var/run/docker.sock:/var/run/docker.sock"
```

Comment:
Do we need to bind the sock? I would prefer not to as I am not a fan of binding absolute paths.

From the docs: 
> Mounting the Docker socket /var/run/docker.sock as a volume is required for the Lambda service.

So I think it's safe to remove this as it's not using the lambda service? I just ran the tests locally without this line and it seems to be fine.

Reply:
Sounds good to me! this is my fault as I was using this container for other task and had lambda included. Let me remove it.